ops/jenkins/bfd-run-pipeline-oob/Jenkinsfile

```diff
@@ -0,0 +1,177 @@
+pipeline {
+  agent {
+    kubernetes {
+      defaultContainer 'bfd-cbc-build'
+      yaml """
+apiVersion: v1
+kind: Pod
+spec:
+  serviceAccount: bfd
+  restartPolicy: Never
+  containers:
+    - name: bfd-cbc-build
+      image: "public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk17-mvn3-tfenv3-latest" # TODO: consider a smarter solution for resolving this image
+      command:
+        - cat
+      tty: true
+      imagePullPolicy: IfNotPresent
+"""
+    }
+  }
+
+  parameters {
+    string(
+      name: 'env',
+      description: 'The BFD Environment to run the BFD Pipeline out-of-band in'
+    )
+    choice(
+      name: 'pipeline_variant',
+      // TODO: Update with RDA when possible
+      choices: ['ccw'],
+      description: 'The variant of BFD Pipeline to run out-of-band'
+    )
+    string(
+      name: 'timeout_minutes',
+      defaultValue: '60',
+      description: 'The timeout, in minutes, that this Jenkin Pipeline will wait until '
+                   + 'automatically scaling-in the out-of-band BFD Pipeline instance'
+    )
```

Comment:
Is this assuming the instance is idle, or a hard limit? I know we have loads that can sometimes take longer than 60 mins.

Reply:
This is a safeguard just in case the operator starting the Jenkins Pipeline steps away for an extended period and forgets about it running. It's configurable, so the operator can specify any timeout they want, I just defaulted it to an hour.